runtime/vm/simulator_arm.cc

```diff
@@ -3040,15 +3040,15 @@ void Simulator::DecodeSIMDDataProcessing(Instr* instr) {
                (instr->Bit(23) == 0) && (instr->Bits(25, 3) == 1)) {
       // Format(instr, "vshlqu'sz 'qd, 'qm, 'qn");
       // Format(instr, "vshlqi'sz 'qd, 'qm, 'qn");
-      const bool signd = instr->Bit(24) == 0;
+      const bool is_signed = instr->Bit(24) == 0;
```

Comment:
According to [Effective dart](https://dart.dev/guides/language/effective-dart/documentation#prefer-starting-a-boolean-variable-or-property-comment-with-whether-followed-by-a-noun-or-gerund-phrase), it should be `isSigned`.

However, the owner my prefer `signed`.